src/unlimited.cpp

```diff
@@ -46,15 +46,15 @@ bool IsTrafficShapingEnabled();
 
 bool MiningAndExcessiveBlockValidatorRule(const unsigned int newExcessiveBlockSize, const unsigned int newMiningBlockSize)
 {
-    // The mined block size must not be greater then the excessive block size.
-    return (newExcessiveBlockSize < newMiningBlockSize);
+    // The mined block size must be less then or equal too the excessive block size.
```

Comment:
the comment is in contradiction with the code (specifically the "or equal to" part)